src/main/java/au/com/rayh/XCodeBuilder.java

```diff
@@ -212,7 +218,7 @@ public XCodeBuilder(Boolean buildIpa, Boolean generateArchive, Boolean cleanBefo
     		String keychainName, String keychainPath, String keychainPwd, String symRoot, String xcodeWorkspaceFile,
     		String xcodeSchema, String buildDir, String developmentTeamName, String developmentTeamID, Boolean allowFailingBuildResults,
     		String ipaName, Boolean provideApplicationVersion, String ipaOutputDirectory, Boolean changeBundleID, String bundleID,
-    		String bundleIDInfoPlistPath, String ipaManifestPlistUrl, Boolean interpretTargetAsRegEx, String ipaExportMethod) {
+    		String bundleIDInfoPlistPath, String ipaManifestPlistUrl, Boolean interpretTargetAsRegEx, String ipaExportMethod, Boolean executeTests, Boolean skipBuild) {
```

Comment:
For migration/upgrade purpose AFAIK you should create a copy of the existing construction with @Deprecated like it was done for others